.github/workflows/python_release.yml

```diff
@@ -55,3 +55,32 @@ jobs:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: .github/workflows/scripts/release_linux.sh
+
+  deploy_windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Setup Bazel
+        uses: abhinavsingh/setup-bazel@v3
```

Comment:
Why is that needed? The Windows image seems to have Bazel included:
https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md